support-frontend/app/controllers/DigitalSubscription.scala

```diff
@@ -59,8 +62,13 @@ class DigitalSubscription(
     val productPrices = priceSummaryServiceProvider.forUser(false).getPrices(DigitalPack, promoCode)
 
     Ok(views.html.main(
-      title, id, js, css, description, canonicalLink, hrefLangLinks, productPrices = Some(productPrices)
-    )).withSettingsSurrogateKey
+      title, id, js, css, description, canonicalLink, hrefLangLinks
+    ) {
+      val priceLines = productPrices.map(p =>
+        s"window.guardian.productPrices = ${outputJson(p)}"
+      )
+      Html(s"""<script type="text/javascript">${priceLines.mkString("\n")}</script>""")
+    }).withSettingsSurrogateKey
```

Comment:
this has been made much more general now in "main" template, so the html is generated here.  Not sure if that's really the correct separation but it's better than before.  But suggestions would be appreciated as I was fighting play a little
This is what it spits into the `/digital` page which looks right.
```html
	<script type="text/javascript">window.guardian.productPrices = ["United Kingdom",{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99}},"Annual":{"GBP":{"price":119.9}}}}}]
window.guardian.productPrices = ["Europe",{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"EUR":{"price":14.99}},"Annual":{"EUR":{"price":149.9}}}}}]
window.guardian.productPrices = ["New Zealand",{}]
window.guardian.productPrices = ["Canada",{}]
window.guardian.productPrices = ["Australia",{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"AUD":{"price":21.5}},"Annual":{"AUD":{"price":215}}}}}]
window.guardian.productPrices = ["International",{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99},"USD":{"price":19.99}},"Annual":{"GBP":{"price":119.9},"USD":{"price":199.9}}}}}]
window.guardian.productPrices = ["United States",{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"USD":{"price":19.99}},"Annual":{"USD":{"price":199.9}}}}}]</script>
```

Reply:
This is not right the json should be:
```
window.guardian.productPrices = {"United Kingdom":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99}},"Annual":{"GBP":{"price":119.9}}}}},"Europe":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"EUR":{"price":14.99}},"Annual":{"EUR":{"price":149.9}}}}},"New Zealand":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"NZD":{"price":23.5}},"Annual":{"NZD":{"price":235}}}}},"Canada":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"CAD":{"price":21.95}},"Annual":{"CAD":{"price":219.5}}}}},"Australia":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"AUD":{"price":21.5}},"Annual":{"AUD":{"price":215}}}}},"International":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99},"USD":{"price":19.99}},"Annual":{"GBP":{"price":119.9},"USD":{"price":199.9}}}}},"United States":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"USD":{"price":19.99}},"Annual":{"USD":{"price":199.9}}}}}}
```

Reply:
ooh glad you checked that then and yes it should have been obvious if I'd looked carefully 🤦‍♂️

Reply:
now I've pushed a change to make it this (looks like NZ and Canada are missing in dev?)
```html
    <script type="text/javascript">window.guardian.productPrices = {"United Kingdom":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99}},"Annual":{"GBP":{"price":119.9}}}}},"Europe":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"EUR":{"price":14.99}},"Annual":{"EUR":{"price":149.9}}}}},"New Zealand":{},"Canada":{},"Australia":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"AUD":{"price":21.5}},"Annual":{"AUD":{"price":215}}}}},"International":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"GBP":{"price":11.99},"USD":{"price":19.99}},"Annual":{"GBP":{"price":119.9},"USD":{"price":199.9}}}}},"United States":{"NoFulfilmentOptions":{"NoProductOptions":{"Monthly":{"USD":{"price":19.99}},"Annual":{"USD":{"price":199.9}}}}}}</script>
```

Reply:
Yes it looks like they are on this page. We'll need to investigate because they are present on the digipack checkout page 😕. Nothing to do with this PR though.